Sources/Honeycomb/HoneycombNavigationSpanInstrumentation.swift

```diff
@@ -0,0 +1,115 @@
+import OpenTelemetryApi
+import OpenTelemetrySdk
+import SwiftUI
+
+private let navigationInstrumentationName = "@honeycombio/instrumentation-navigation"
+private let navigationSpanName = "Navigation"
+private let unencodablePath = "<unencodable path>"
+
+func getTracer() -> Tracer {
+    return OpenTelemetry.instance.tracerProvider.get(
+        instrumentationName: navigationInstrumentationName,
+        instrumentationVersion: honeycombLibraryVersion
+    )
+}
+
+internal class HoneycombNavigationProcessor {
+    static let singleton = HoneycombNavigationProcessor()
+    var currentNavigationPath: String? = nil
+
+    private init() {}
+
+    @available(iOS 16.0, macOS 12.0, *)
+    func reportNavigation(path: NavigationPath) {
+        if let codablePath = path.codable {
+            reportNavigation(path: codablePath)
+        } else {
+            reportNavigation(path: unencodablePath)
+        }
+    }
+
+    func reportNavigation(path: String) {
+        print("current path: \(path)")
```

Comment:
I think this was just for debug and can be removed?

Reply:
yeah